applications/nrf_desktop/src/modules/Kconfig.config_channel

```diff
@@ -78,6 +78,21 @@ config DESKTOP_CONFIG_CHANNEL_DFU_SYNC_BUFFER_SIZE
 	  The host must perform progress synchronization at least
 	  every synchronization buffer bytes count.
 
+config DESKTOP_CONFIG_CHANNEL_DFU_MCUBOOT_XIP
+	bool "Device uses MCUboot bootloader with direct XIP support"
+	depends on BOOTLOADER_MCUBOOT
+	default y if BOOT_BUILD_DIRECT_XIP_VARIANT
+	help
+	  The option informs DFU module that the MCUboot bootloader supports
+	  direct execute-in-place (XIP). In XIP mode, the image is booted
+	  directly from secondary slot instead of moving it to the primary
+	  slot. The module by default assumes that MCUboot with XIP is used when
+	  direct XIP variant image is built.
+
+	  The DFU module assumes that MCUboot XIP bootloader simply boots image
+	  with higher version. The module does not mark the newly uploaded image
+	  as pending nor confirm it after a successful boot.
```

Comment:
```suggestion
	  The option informs the DFU module that the MCUboot bootloader supports
	  direct-xip (execute-in-place). In this mode mode, the image is booted
	  directly from the secondary slot instead of moving it to the primary
	  slot. The module by default assumes that MCUboot with XIP is used when
	  direct-xip variant image is built.

	  The DFU module assumes that the MCUboot direct-xip bootloader simply boots the image
	  with a higher version. The module does not mark the newly uploaded image
	  as pending nor confirm it after a successful boot.
```

Reply:
Added